jobs/release.Jenkinsfile

```diff
@@ -275,6 +282,39 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
             }
         }
 
+        if (brew_profile) {
+            stage('Brew Upload') {
+                def tag = pipecfg.streams[params.STREAM].brew_tag
+                for (arch in basearches) {
+                    def state = false
+                    def version = (params.VERSION).replaceAll("-", ".")
```

Comment:
weird. why is this needed?

Reply:
Because we do this replace [here](https://github.com/coreos/coreos-assembler/blob/main/src/cmd-koji-upload#L437). What I think make sense to have the rhcos version without '-' in the NVR name. I think we can also remove this replace if we want to, but need to see with koji allow it too, but I guess so.

Reply:
I don't think these "replacements" are doing anything in either python or groovy. The first argument is the string you want to replace and the second is the string you want to replace it with.

The buildid already starts out as `413.92.202305261801.0` so there are no `-` characters to replace IIUC. You can see this in the brew upload you shared with me earlier: https://brewweb.stage.engineering.redhat.com/brew/buildinfo?buildID=2292952

Reply:
That's because it got changed  in the link I sent above before adding it to Brew, if you see the parameters from Jenkins, the Version ID is like:  '413.92.202305261801-0' and with that replace we make it as '413.92.202305261801.0'

Reply:
oh I see. because of https://github.com/openshift/os/blob/64e21bb5eedf19cdc0b16db235e4d24ca5e86ecf/manifest-rhel-9.2.yaml#L29

so our versions are like:

```
414.92.202307122109-0
```

but it's interesting to note that the version string is [down to the minute](https://github.com/openshift/os/blob/64e21bb5eedf19cdc0b16db235e4d24ca5e86ecf/manifest-rhel-9.2.yaml#L27) which virtually guarantees there will never be anything other than `-0` in the version string. I wonder if we should pivot what we are doing in https://github.com/coreos/coreos-assembler/pull/3533 to make it such that VR represents the rhcos buildid.

i.e.:

```
$ V=414.92.202307122109
$ R=-0
$ echo "${V}${R}"
414.92.202307122109-0
```

This would make it a bit more intuitive I think.